config/initializers/notifications.rb

```diff
@@ -0,0 +1,10 @@
+# rubocop:disable Metrics/LineLength
+# rubocop:disable Style/FormatStringToken
+ActiveSupport::Notifications.subscribe('request.faraday') do |_name, start_time, end_time, _id, env|
+  url = env[:url]
+  http_method = env[:method].to_s.upcase
+  duration = end_time - start_time
+  STDOUT.puts '[%s] %s %s (%.3f s)' % [url.host, http_method, url.request_uri, duration]
```

Comment:
Did rubocop not complain about not using ""?  Also might it best to use the rails logger if possible?

Reply:
Updated to use Rails logger - no complaints from Rubocop about single quotes - maybe it's ok for this type of string formatting?